### PR TITLE
Added a prop to be able to crop the item filter on the right or on the left

### PR DESCRIPTION
**Pull request by sroy3**
screen shot 2017-02-10 at 3 11 50 pm

### CODE REVIEW

**Comment by MichelML** [comment]
in what case would you use a left crop? are we planning to use it? that's nice though, just wondering in what circumstances it would be utilized 😄

**Comment by sroy3** [comment]
@MichelML actually in the log browser, we will be cropping on the left to see the end of the uri.

File: src/components/actions/filters/ItemFilter.tsx [hunk 1]
```diff
@@ -5,15 +5,29 @@ export interface IItemFilterProps extends React.ClassAttributes<ItemFilter> {
...
 export class ItemFilter extends React.Component<IItemFilterProps, any> {
 
   render() {
+    let itemFilter: string = this.props.item;
+    if (this.props.crop) {
+      let itemFilterLength: number = itemFilter.length;
+
+      if (itemFilterLength > Math.abs(this.props.crop)) {
+        itemFilter = this.props.crop > 0
+          ? itemFilter.substring(0, this.props.crop) + ELLIPSIS
```
Comment (pochretien) [inline src/components/actions/filters/ItemFilter.tsx]: You can use s.truncate(itemFilter, this.props.crop) with underscore strings, just another way to add the ellipsis. We can add third param to custom the ellipsis if we want :) but default is ...

Comment (sroy3) [inline src/components/actions/filters/ItemFilter.tsx]: I wish it did work, but it only truncates on one side. Better keep consistency between cropping right and cropping left and use substring.

**Pull request merged**